fMRIVolume/GenericfMRIVolumeProcessingPipeline.sh

```diff
@@ -287,6 +320,10 @@ T1wFolder="$Path"/"$Subject"/"$T1wFolder"
 AtlasSpaceFolder="$Path"/"$Subject"/"$AtlasSpaceFolder"
 ResultsFolder="$AtlasSpaceFolder"/"$ResultsFolder"/"$NameOffMRI"
 
+if [ ! $fMRIReference = "NONE" ] ; then
```

Comment:
When this condition is true, shouldn't it trigger other downstream changes in terms of the "Scout" treatment?

Reply:
The idea is to use the actual BOLD scout for other functionality and use the reference scout exclusively as a registration target.